src/FieldWarning/Assets/Model/Armory/Configs.cs

```diff
@@ -16,6 +16,27 @@
 
 namespace PFW.Model.Armory
 {
+    // TODO make this not be a copy of deck..
+    [Serializable]
+    public class ArmoryConfig
```

Comment:
Also i'm thinking Armory should maybe be like a 'factory' that builds all these units?